lib/capistrano/tasks/capistrano-resque-pool.rake

```diff
@@ -18,7 +18,15 @@ namespace :resque do
     desc 'Gracefully shut down workers and shutdown the manager after all workers are done'
     task :stop do
       on roles(workers) do
-        execute :kill, "-s QUIT `cat #{pid_path}`"
+        if test("[ -f #{pid_path} ]")
```

Comment:
What do you think about moving it to separate method (e.g. `pid_file_exists?`) since we use it in few places?